lib/routes/bing/daily-wallpaper.ts

```diff
@@ -1,41 +1,72 @@
 import { Route } from '@/types';
-import ofetch from '@/utils/ofetch';
+import cache from '@/utils/cache';
+import got from '@/utils/got';
 import { parseDate } from '@/utils/parse-date';
 import timezone from '@/utils/timezone';
 
 export const route: Route = {
-    path: '/',
+    path: '/:routeParams?',
+    parameters: {
+        routeParams: '额外参数type和story；请参阅以下说明和表格',
+    },
     radar: [
         {
             source: ['cn.bing.com/'],
             target: '',
         },
     ],
     name: '每日壁纸',
-    maintainers: ['FHYunCai'],
+    maintainers: ['FHYunCai', 'LLLLLFish'],
     handler,
     url: 'cn.bing.com/',
+    example: '/bing/type=UHD&story=1',
+    description: `| 参数    | 含义        | 接受的值                                                      | 默认值       | 备注                                                     |
+|-------|-----------|-----------------------------------------------------------|-----------|--------------------------------------------------------|
+| type  | 输出图片的像素类型 | UHD/1920x1080/1920x1200/768x1366/1080x1920/1080x1920_logo | 1920x1080 | 1920x1200与1080x1920_logo带有水印,输入的值不在接受范围内都会输出成1920x1080 |
+| story | 是否输出图片的故事 | 1/0                                                       | 0         | 输入的值不为1都不会输出故事                                         |
+`,
 };
 
 async function handler(ctx) {
-    const response = await ofetch('HPImageArchive.aspx', {
-        baseURL: 'https://cn.bing.com',
-        query: {
-            format: 'js',
-            idx: 0,
-            n: ctx.req.query('limit') ? Number.parseInt(ctx.req.query('limit'), 10) : 7,
-            mkt: 'zh-CN',
-        },
+    const routeParams = new URLSearchParams(ctx.req.param('routeParams'));
+    const lang = 'zh-CN';
+    let type = routeParams.get('type') || '1920x1080';
+    const allowedTypes = ['UHD', '1920x1080', '1920x1200', '768x1366', '1080x1920', '1080x1920_logo'];
+    if (!allowedTypes.includes(type)) {
+        type = '1920x1080';
+    }
+    const story = routeParams.get('story') === '1';
+    const apiUrl = `https://cn.bing.com/hp/api/model`;
```

Comment:
```suggestion
    const apiUrl = 'https://cn.bing.com/hp/api/model';
```
Unnecessary use of template literal.